src/main/java/pl/javastart/task/ToDoTask.java

```diff
@@ -0,0 +1,68 @@
+package pl.javastart.task;
+
+public class ToDoTask {
+
+    private String name;
+    private String description;
+    private int priority;
+    private Person person;
+
+    public ToDoTask(String name, String description) {
+        this.name = name;
+        this.description = description;
+    }
+
+    public ToDoTask(String name, String description, int priority) {
+        this.name = name;
+        this.description = description;
+        this.priority = priority;
+    }
+
+    public ToDoTask(String name, String description, int priority, Person person) {
+        this.name = name;
+        this.description = description;
+        this.priority = priority;
+        this.person = person;
+    }
+
+    boolean highPriority() {
+        return (this.priority > 0);
+    }
+
+    boolean mediumPriority() {
+        return (this.priority == 0);
+    }
+
+    boolean lowPriority() {
+        return (this.priority < 0);
+    }
+
+    void showTaskPriority() {
+        System.out.print("Zadanie " + this.name + " ma priorytet ");
+        String p = "średni";
+        if (this.lowPriority()) { p = "niski"; }
+        if (this.highPriority()) { p = "wysoki"; }

```

Comment:
Jeżeli w ifie jest nawet tylko jeden wiersz, to zapisuj to z rozbiciem na wiersze:
```
if () {
   //..kod
}
```